uk.co.compucorp.civicrm.hrleaveandabsences/settings/leaveandabsence.setting.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+  return [
+    'relationship_allowed_to_approve_leave' => [
+      'group_name' => 'Leave and Absence General Settings',
+      'group' => 'leaveandabsence',
```

Comment:
what is the difference between group and group_name? Do they represent the same thing (as in, group_name is the display name and group is the internal machine name)? If so, maybe group should be renamed to `leave_an_absences_general_settings`?